Form/DataTransformer/ModelsToArrayTransformer.php

```diff
@@ -23,14 +23,21 @@
  */
 class ModelsToArrayTransformer implements DataTransformerInterface
 {
+    /**
+     * @var object
+     */
     protected $choiceList;
 
     /**
-     * @param \Sonata\AdminBundle\Form\ChoiceList\ModelChoiceList $choiceList
+     * @param object $choiceList
      */
-    public function __construct(ModelChoiceList $choiceList)
+    public function __construct($choiceList)
```

Comment:
isn't there an interface which is used by both of the ```choiceLists```?

completely removing the type hint is not so good IMO